packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -456,8 +456,16 @@ export class OdspDocumentService implements IDocumentService {
         }
     }
 
-    public dispose() {
-        this._opsCache?.flushOps();
+    public dispose(criticalError: boolean) {
```

Comment:
dispose already has an overload to take an error. i don't really like this non-standard overload: https://github.com/microsoft/FluidFramework/blob/02bd5378e75ab0c28b13f63e702e740365fda321/common/lib/common-definitions/src/disposable.ts#L8